prog/rand.go

```diff
@@ -17,7 +17,7 @@ import (
 
 const (
 	// "Recommended" number of calls in programs that we try to aim at during fuzzing.
-	RecommendedCalls = 20
+	RecommendedCalls = 30
```

Comment:
It looks like a global change that could significantly affect syzbot performance.

Reply:
Yes, it can.
Note it used to be 30 in the past:
https://github.com/google/syzkaller/commit/9b1f3e665308